### PR TITLE
[FIX] components: fix button/input styles

### PR DESCRIPTION
**Pull request by rrahir**
## Description:

description of this task, what is implemented and why it is implemented that way.

Task: : [TASK_ID](https://www.odoo.com/web#id=TASK_ID&action=333&active_id=2328&model=project.task&view_type=form&cids=1&menu_id=4720)

## review checklist

- [ ] feature is organized in plugin, or UI components
- [ ] support of duplicate sheet (deep copy)
- [ ] in model/core: ranges are Range object, and can be adapted (adaptRanges)
- [ ] in model/UI: ranges are strings (to show the user)
- [ ] undo-able commands (uses this.history.update)
- [ ] multiuser-able commands (has inverse commands and transformations where needed)
- [ ] new/updated/removed commands are documented
- [ ] exportable in excel
- [ ] translations (\_t("qmsdf %s", abc))
- [ ] unit tested
- [ ] clean commented code
- [ ] track breaking changes
- [ ] doc is rebuild (npm run doc)
- [ ] status is correct in Odoo

### CODE REVIEW

File: src/components/grid_add_rows_footer/grid_add_rows_footer.xml [hunk 1]
```diff
@@ -4,7 +4,12 @@
       class="o-grid-add-rows mt-2 ms-2 w-100 d-flex position-relative align-items-center"
       t-att-style="addRowsPosition"
       t-on-mousedown.stop.prevent="">
-      <button t-on-click="onConfirm" t-att-disabled="state.errorFlag" class="o-button">Add</button>
+      <button
+        t-on-click="onConfirm"
+        t-att-disabled="state.errorFlag"
+        class="o-button o-button-grey">
```
Comment (hokolomopo) [inline src/components/grid_add_rows_footer/grid_add_rows_footer.xml:10]: If every single button with the class `o-button` should also have the class `o-button-grey`, what's the point of the class `o-button-grey` ? Just includes it in the class `o-button`. Just make it so `btn-outline-danger` correctly overwrites `o-button`

**Review by hokolomopo — COMMENTED** [review]
Commit message is missing task id 🙂 

Odoo's runbot is super red btw

File: src/components/grid_add_rows_footer/grid_add_rows_footer.xml (hunk 1 above)
Comment (rrahir) [reply src/components/grid_add_rows_footer/grid_add_rows_footer.xml:10]: I chose to go with the same approach Bootstrap seems to have , split the form from the colors which is better if we want to have a clean support of different color schemes in the future.
The runbot "red" has nothing to do with this PR. I started a rebuild and we'll see

**Comment by rrahir** [comment]
@robodoo rebase-ff

**Pull request closed without merging**